sdks/java/testing/tpcds/build.gradle

```diff
@@ -16,38 +16,106 @@
  * limitations under the License.
  */
 
-plugins {
-    id 'java'
-}
+plugins { id 'org.apache.beam.module' }
+applyJavaNature(
+        automaticModuleName: 'org.apache.beam.sdk.tpcds',
+        exportJavadoc: false,
+        archivesBaseName: 'beam-sdks-java-tpcds',
+)
 
-description = "Apache Beam :: SDKs :: Java :: TPC-DS Benchark"
+description = "Apache Beam :: SDKs :: Java :: TPC-DS"
 
-version '2.24.0-SNAPSHOT'
+// When running via Gradle, this property can be used to pass commandline arguments
+// to the TPD-DS run
+def tpcdsArgsProperty = "tpcds.args"
 
-sourceCompatibility = 1.8
+// When running via Gradle, this property sets the runner dependency
+def tpcdsRunnerProperty = "tpcds.runner"
+def tpcdsRunnerDependency = project.findProperty(tpcdsRunnerProperty)
+        ?: ":runners:direct-java"
+def shouldProvideSpark = ":runners:spark:2".equals(tpcdsRunnerDependency)
+def isDataflowRunner = ":runners:google-cloud-dataflow-java".equals(tpcdsRunnerDependency)
+def runnerConfiguration = ":runners:direct-java".equals(tpcdsRunnerDependency) ? "shadow" : null
+
+if (isDataflowRunner) {
```

Comment:
@kennknowles I'm not sure about the purpose of this test but do we consider adding runner_v2 as well?

Reply:
It is a standard benchmark suite. Yes, it makes sense to run on runner_v2 (and any other runner configuration someone is interested in)